### DISCUSSION TITLE
GStreamer Error While Streaming to AWS Kinesis on Raspberry Pi

### DISCUSSION THREAD

**Discussion by ambroseeverhart** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#1862] [Q&A]
I'm diving into the world of streaming with GStreamer on a Raspberry Pi to AWS Kinesis, and I've hit a roadblock. I'm encountering the following error:

(kvsWebrtcClientMasterGstSample:246): GStreamer-CRITICAL **: gst_bus_timed_pop_filtered: assertion 'types != 0' failed

i gave correct access key ,secret key ,all dependencies, credentials and certificate path but the issue is persistent

Has anyone else faced this issue? If so, could you share some insights on how to tackle it? I'm eager to get past this hurdle and continue with my streaming project.

Thanks in advance for any help or suggestions!

**Comment by stefankiesz** [comment]
Hi Ambrose, were you able to get past this issue?
